tests/rest_api/test_configurations.py

```diff
@@ -600,10 +612,11 @@ def test_constructs_incremental_from_endpoint_config_incremental():
             "end_param": "until",
             "cursor_path": "updated_at",
             "initial_value": "2024-01-25T11:21:28Z",
+            "transform": lambda epoch: pendulum.from_timestamp(int(epoch)),
```

Comment:
We may also need to keep the case where `transform` is missing or None.

Reply:
Good idea! Now we test with a callable argument and without.